drivers/video/ov5640.c

```diff
@@ -325,6 +334,114 @@ static const struct ov5640_reg ov5640InitParams[] = {
 	{0x5000, 0xa7},
 };
 
+/* FIXME: Check with values are really necessary. */
```

Comment:
Just my impression: Hard to know for each of them, and also true for the other parameters above, so might be possible to remove that comment, with more pull requests to characterize more of these registers.

Reply:
Done.